message/validation/genesis/validation.go

```diff
@@ -46,7 +45,7 @@ const (
 	clockErrorTolerance = time.Millisecond * 50
 
 	maxMessageSize             = maxConsensusMsgSize
-	maxConsensusMsgSize        = 8388608
+	maxConsensusMsgSize        = 6291829
```

Comment:
does this also relates to post-conesnsus messages? afaik they got bigger

Reply:
It's the latest value from stage: https://github.com/ssvlabs/ssv/blob/stage/message/validation/validation.go#L47

Reply:
@MatheusFranco99 doesn't need to be calculated again for CC??